src/processor/operator/order_by/top_k.cpp

```diff
@@ -8,6 +9,18 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace processor {
 
+std::string TopKPrintInfo::toString() const {
+    std::string result = "Keys: ";
+    result += binder::ExpressionUtil::toString(keys);
+    result += ", Expressions: ";
+    result += binder::ExpressionUtil::toString(payloads);
+    result += ", Skip Value: ";
```

Comment:
Could you double check that if you don't include a skip in the query that it shows up as 0

Reply:
I Updated the pictures, is it correct?

Reply:
Yeah works correctly! But come to think of it can we remove value and number and have it say `Skip: ` and `Limit: ` instead of `Skip Value: ` and `Limit Number: `